### PR TITLE
Clean up tutorial titles

### PR DESCRIPTION
**Pull request by bermudezmt**
Fixes https://github.com/pulumi/docs/issues/1670

First attempt at standardizing our example/tutorial titles. Let me know if I missed anything, or if you can come up with better titles. Thanks!

### CODE REVIEW

File: aws-ts-apigateway/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new)
 
-# Serverless REST API on AWS
+# Serverless REST API
```
Comment (justinvp) [inline aws-ts-apigateway/README.md]: Some titles keep "AWS" in the title. Not pushing back, just curious why we're removing it here and in other places "on AWS" is used.

File: aws-ts-static-website/README.md [hunk 2]
```diff
@@ -1,7 +1,6 @@
 [![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new)
 
-# Static Website using AWS and TypeScript
-
+# Secure Static Website Using Amazon S3, CloudFront, Route53, and Certificate Manager
```
Comment (justinvp) [inline aws-ts-static-website/README.md]:
```suggestion
# Secure Static Website Using Amazon S3, CloudFront, Route53, and Certificate Manager

```

File: cloud-ts-voting-app/README.md [hunk 3]
```diff
@@ -1,7 +1,6 @@
 [![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new)
 
-# Voting app with two containers
-
+# Voting App Using Redis and Flask
```
Comment (justinvp) [inline cloud-ts-voting-app/README.md]:
```suggestion
# Voting App Using Redis and Flask

```

File: aws-ts-apigateway/README.md (hunk 1 above)
Comment (bermudezmt) [reply aws-ts-apigateway/README.md]: I was torn on some titles that are really short like our basic examples for provisioning resources. It made sense to leave the cloud provider in, but the directory structure (in the repo) and the breadcrumbs will probably make this redundant. Might be best to just drop "AWS", "Azure", and "GCP" from the titles.

File: aws-js-containers/README.md [hunk 4]
```diff
@@ -1,25 +1,25 @@
...
 To run this example, make sure [Docker](https://docs.docker.com/engine/installation/) is installed and running.
 
-## Running the App
+## Deploy the App
 
 Note: some values in this example will be different from run to run.  These values are indicated
 with `***`.
 
-1.  Create a new stack:
+Step 1:  Create a new stack
```
Comment (mikhailshilkov) [inline aws-js-containers/README.md]: Shouldn't "Step X" be a title? ([example](https://www.digitalocean.com/community/tutorials/how-to-create-a-cluster-of-docker-containers-with-docker-swarm-and-digitalocean-on-ubuntu-16-04#step-1-%E2%80%94-provisioning-the-cluster-nodes))

File: aws-js-webserver/README.md [hunk 5]
```diff
@@ -1,6 +1,6 @@
 [![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new)
 
-# AWS EC2 Web Server
+# Python Web Server Using Amazon EC2
```
Comment (mikhailshilkov) [inline aws-js-webserver/README.md]: The previous example is `AWS Web Server Component` while this one is `Python Web Server Using Amazon EC2`. They are almost the same thing, so I would structure the title in a similar way.
I would do something like `Web Server (Component) Using Amazon EC2`

Comment (mikhailshilkov) [inline aws-js-webserver/README.md]: Why Python - that's a `aws-js-x` example?

File: aws-py-webserver/README.md [hunk 6]
```diff
@@ -1,6 +1,6 @@
 [![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new)
 
-# AWS Web Server example in Python
+# Simple Web Server Using Amazon EC2
```
Comment (mikhailshilkov) [inline aws-py-webserver/README.md]: Why qualify it as simple? Were other Web Servers complex?

File: aws-ts-apigateway/README.md (hunk 1 above)
Comment (mikhailshilkov) [reply aws-ts-apigateway/README.md]: Could the rule be "it's okay to add names of specific services (S3, EC2), but don't use blank 'AWS'"?

File: aws-ts-eks-migrate-nodegroups/README.md [hunk 7]
```diff
@@ -1,8 +1,9 @@
-# examples/aws-ts-eks-migrate-nodegroups
+# Migrating EKS Node Groups with Zero Downtime
```
Comment (mikhailshilkov) [inline aws-ts-eks-migrate-nodegroups/README.md]: Should we standardize on using imperatives ("Host a website") or gerunds ("Migrating EKS")?

File: aws-ts-ruby-on-rails/README.md [hunk 8]
```diff
@@ -1,6 +1,6 @@
 [![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new)
 
-# AWS EC2 Ruby on Rails
+# Ruby on Rails Server Using Amazon EC2
```
Comment (mikhailshilkov) [inline aws-ts-ruby-on-rails/README.md]: Why "**on** AWS Fargate" but "**Using** Amazon EC2"? I'm sure you have a good reason but I don't know it yet :)

File: aws-ts-voting-app/README.md [hunk 9]
```diff
@@ -1,6 +1,6 @@
 [![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new)
 
-# Voting app with two containers
+# Voting app Using Reeis and Flask
```
Comment (mikhailshilkov) [inline aws-ts-voting-app/README.md]: Redis

File: azure-js-webserver/README.md [hunk 10]
```diff
@@ -1,6 +1,6 @@
 [![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new)
 
-# Pulumi web server (Azure)
+# Pulumi Web Server
```
Comment (mikhailshilkov) [inline azure-js-webserver/README.md]: Web Server Using Azure Virtual Machine?

File: azure-py-webserver/README.md [hunk 11]
```diff
@@ -1,6 +1,6 @@
 [![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new)
 
-# Azure Web Server example in Python
+# Python Web Server Using Azure Virtual Machine
```
Comment (mikhailshilkov) [inline azure-py-webserver/README.md]: Remove "Python"?

File: azure-ts-appservice-devops/README.md [hunk 12]
```diff
@@ -1,6 +1,6 @@
 [![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new?template=https://github.com/pulumi/examples/tree/master/azure-ts-appservice-devops/infra)
 
-# A Todo App on Azure App Service with SQL Database and Application Insights and deploys it to Azure DevOps
+# Todo App with SQL Database and Application Insights Integrated with Azure DevOps
```
Comment (mikhailshilkov) [inline azure-ts-appservice-devops/README.md]: App Service is probably more relevant than e.g. Application Insights to stay in the title

File: azure-ts-cosmosdb-logicapp/README.md [hunk 13]
```diff
@@ -1,8 +1,8 @@
 [![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new)
 
-# Deploy an Azure Cosmos DB container, an API Connection, and a Logic App
+# Azure Cosmos DB container, an API Connection, and a Logic App
```
Comment (mikhailshilkov) [inline azure-ts-cosmosdb-logicapp/README.md]: I'd remove the word "container"

File: azure-ts-functions-raw/README.md [hunk 14]
```diff
@@ -1,6 +1,6 @@
 [![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new)
 
-# Azure Functions
+# Azure Functions in .NET and Java
```
Comment (mikhailshilkov) [inline azure-ts-functions-raw/README.md]: There are like six languages in there. List them all or replace with "all supported languages"?

File: azure-ts-webserver-component/README.md [hunk 15]
```diff
@@ -1,6 +1,6 @@
 [![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new)
 
-# Azure Web Server Virtual Machine Component
+# Azure Web Server Component
```
Comment (mikhailshilkov) [inline azure-ts-webserver-component/README.md]: Web Server Component Using Azure Virtual Machine?

File: cloud-js-containers/README.md [hunk 16]
```diff
@@ -1,6 +1,6 @@
 [![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new)
 
-# Easy container example
+# Containers on AWS Fargate
```
Comment (mikhailshilkov) [inline cloud-js-containers/README.md]: Isn't this supposed to be cloud-agnostic?

File: gcp-js-webserver/README.md [hunk 17]
```diff
@@ -1,6 +1,6 @@
 [![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new)
 
-# Pulumi web server (GCP)
+# Web Server in Google Cloud
```
Comment (mikhailshilkov) [inline gcp-js-webserver/README.md]: Web Server Using Compute Engine? (I believe that's the name of "GCP's EC2"?)

File: gcp-py-functions/README.md [hunk 18]
```diff
@@ -1,6 +1,6 @@
 [![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new)
 
-# GCP Functions
+# Google Cloud Functions in Python
```
Comment (mikhailshilkov) [inline gcp-py-functions/README.md]: Omit language?

File: gcp-py-instance-nginx/README.md [hunk 19]
```diff
@@ -1,6 +1,6 @@
 [![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new)
 
-# Pulumi Nginx Server in a instance (GCP)
+# Pulumi Nginx Server in Google Cloud
```
Comment (mikhailshilkov) [inline gcp-py-instance-nginx/README.md]: Nginx Server Using Compute Engine?

File: gcp-ts-functions/README.md [hunk 20]
```diff
@@ -1,6 +1,6 @@
 [![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new)
 
-# GCP Functions
+# Google Cloud Functions in TypeScript
```
Comment (mikhailshilkov) [inline gcp-ts-functions/README.md]: Omit language?

File: kubernetes-ts-guestbook/README.md [hunk 21]
```diff
@@ -1,6 +1,6 @@
 [![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new?template=https://github.com/pulumi/examples/tree/master/kubernetes-ts-guestbook/components)
 
-# Kubernetes Guestbook (Two Ways)
+# Simple and Component-based Kubernetes Guestbook Apps
```
Comment (mikhailshilkov) [inline kubernetes-ts-guestbook/README.md]: Again, why "simple"?

File: kubernetes-ts-helm-wordpress/README.md [hunk 22]
```diff
@@ -1,6 +1,6 @@
 [![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new)
 
-# Kubernetes: Deploying the Wordpress Helm chart
+# Deploying the Wordpress Helm Chart
```
Comment (mikhailshilkov) [inline kubernetes-ts-helm-wordpress/README.md]: Omit "deploying"?

File: linode-js-webserver/README.md [hunk 23]
```diff
@@ -1,6 +1,6 @@
 [![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new)
 
-# Pulumi web server (Linode)
+# Pulumi Web Server on Linode
```
Comment (mikhailshilkov) [inline linode-js-webserver/README.md]: Omit "Pulumi"?

File: multicloud-ts-buckets/README.md [hunk 24]
```diff
@@ -1,6 +1,6 @@
 [![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new)
 
-# Multi-Cloud in AWS and GCP using TypeScript
+# AWS and GCP Resources Using TypeScript
```
Comment (mikhailshilkov) [inline multicloud-ts-buckets/README.md]: Omit "Using TypeScript"?

File: aws-ts-apigateway/README.md (hunk 1 above)
Comment (bermudezmt) [reply aws-ts-apigateway/README.md]: Yes, that is correct. I'll clarify that in the example template.

File: aws-js-webserver/README.md (hunk 5 above)
Comment (bermudezmt) [reply aws-js-webserver/README.md]: It is running a Python web server but the program is written in JS. Either way, I've changed to follow the other title structure. Thanks for spotting this!

File: aws-ts-eks-migrate-nodegroups/README.md (hunk 7 above)
Comment (bermudezmt) [reply aws-ts-eks-migrate-nodegroups/README.md]: I like imperatives, but our website uses gerunds a lot. I've avoided using verbs here. Joe changed this to "Zero Downtime Migration" on the site so I'll lead with that.

File: aws-ts-ruby-on-rails/README.md (hunk 8 above)
Comment (bermudezmt) [reply aws-ts-ruby-on-rails/README.md]: I think I just got lazy. :) Let's change to "Using" on the Fargate one.

File: cloud-js-containers/README.md (hunk 16 above)
Comment (bermudezmt) [reply cloud-js-containers/README.md]: Good catch! Although it explicitly states it's a companion to the AWS tutorial. What was the motivation behind `cloud` examples? The others do explicitly use the `pulumi-cloud` package. @justinvp should we add that these examples are using a preview library?

File: gcp-js-webserver/README.md (hunk 17 above)
Comment (bermudezmt) [reply gcp-js-webserver/README.md]: THANK YOU.

File: kubernetes-ts-guestbook/README.md (hunk 21 above)
Comment (bermudezmt) [reply kubernetes-ts-guestbook/README.md]: In the actual example, it mentions showing the Guestbook "two ways", and calls the variants these:
```
simple/ is a straight port of the original YAML.
components demonstrates benefits of using a real language, namely eliminating boilerplate through the use of real component abstractions.
```

Comment (bermudezmt) [reply kubernetes-ts-guestbook/README.md]: Open to your alternative suggestion but leaving it as is for now

**Pull request merged**